README.md

```diff
@@ -222,6 +223,9 @@ Building this provider requires a 64 bit Linux or MacOS build system with the fo
 * C++ build chain
 * [lcov](http://ltp.sourceforge.net/coverage/lcov.php) for coverage metrics
 * [gcovr](https://gcovr.com/en/stable/) for reporting coverage metrics in CodeBuild
+* [Go](https://golang.org/dl/) 1.18 or later is required. If not found by 
```

Comment:
This isn't currently documented in AWS-LC's documentation, but Go v1.20 or later should be used when possible. 
ninja-build: fine with Go 1.18
ninja-build run_test: sorry, need Go 1.20

I'd change this line to `1.18 or later is the minimum required version to build AWS-LC, 1.20 or later is needed in order to run AWS-LC's test suite.`

Reply:
Updated!